### DISCUSSION TITLE
Tuxguitar Flatpak?

### DISCUSSION THREAD

**Discussion by alexisguerreroabg** [helge17/tuxguitar#405] [General]
Hi, any chance of updating a flatpak version?, the flathub version is 1.6.2

Thanks.

**Comment by guiv42** [comment]
Hi
Currently 1.6.2 is the latest stable version, so flatpak is up-to-date!

1.6.3 should be published in a few days. I plan to submit a pull request to update flatpak once it is published. Stay tuned!

**Reply by alexisguerreroabg** [reply]
Sorry for my mistake, I had installed an older version of 1.6.2 that kept fedora from its repositories, not the flathub version, there was my confusion.

Thank you.

**Comment by guiv42** [comment]
@alexisguerreroabg : just for your information, 1.6.3 has been released today, and I created a PR to update the flatpak version. Now it's beyond our control here.
